src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -2,0 +1,29 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { UserInfo } from '../UserInfo';
+import classNames from 'classnames';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const todoOwner: User = usersFromServer.find(
+    (user: User) => user.id === todo.userId,
+  ) as User;
```

Comment:
[REACT KNOWLEDGE] - The component is directly importing and using data from `usersFromServer`, which can lead to unoptimized performance and breaks the single source of truth principle. It's better to lift this logic to a parent component, pass the necessary data via props, and handle conditional rendering there.